packages/content-conformance/src/config.ts

```diff
@@ -0,0 +1,60 @@
+import z from 'zod'
+import { findUp } from 'find-up'
+import path from 'path'
+import { loadModuleFromFilePath } from './utils.js'
+
+const CONFIG_FILE_NAME = 'content-conformance.config.js'
+
+const ContentConformanceConfigRule = z
+  .string()
+  .or(z.tuple([z.string(), z.record(z.any())]))
```

Comment:
💭 Stricter validation, and exporting for usage elsewhere like in `rules.ts`

```suggestion
export const LEVELS = ['off', 'warn', 'error'] as const
export const LevelsEnum = z.enum(LEVELS)
export const ContentConformanceConfigRule = LevelsEnum.or(
  z.tuple([z.string(), z.record(z.any())])
)
```

Reply:
Do you think we want to export the `zod` object for `LevelsEnum` or just the type?

Reply:
I've exported the type for now. Thanks for the suggestion!